recipes/gromacs/meta.yaml

```diff
@@ -2,14 +2,14 @@
 # For latest release, see
 # https://manual.gromacs.org/documentation/
 {% set name = "gromacs" %}
-{% set version = "2022.0" %}
+{% set version = "2022" %}
```

Comment:
Even though the gromacs tar file is labelled 2020, should we still set the version here to 2020.0 (and hard code the url for this download)?

Reply:
No - ignore me - we have followed this convention in the bioconda releases too.